### DISCUSSION TITLE
Using Groops processing data of BeiDou system

### DISCUSSION THREAD

**Discussion by FFFFFG-IGG** [groops-devs/groops#183] [Q&A]
Hello,
  In #76 I've shown that BeiDou performs worse in fixed PPP than float PPP. And @zhedumi mentioned that this issue(BeiDou) was under resolution on their branch from then on. I would like to ask where can I get these updates of groops(may include BeiDou analysis and PPP)? Note that I used products(BeiDou2 BeiDou3) provided by PPP-AR group of WHU(Jianghui Geng) in #76.  
Best regards,
Yongzhao Fan

**Comment by Waley04** [comment]
@FFFFFG-IGG I have also been waiting for such updates.

**Comment by zhedumi** [comment]
Hi,

Sorry for the late reply. This issue is still being investigated since the people working on GNSS amounts to 2 and on the priority list BeiDou fell to the back at the current state. Updates that also should influence PPP BeiDou should also be in the main already but generally more thorough investigations regarding PPP BeiDou especially have been delayed due to other more urgent tasks. I am not sure when I will be able to work more on BeiDou in the close future. I think the earliest I might be able to investigate this more thorough is in August honestly.


Best regards,

Patrick

**Reply by FFFFFG-IGG** [reply]
> Sorry for the late reply. This issue is still being investigated since the people working on GNSS amounts to 2 and on the priority list BeiDou fell to the back at the current state. Updates that also should influence PPP BeiDou should also be in the main already but generally more thorough investigations regarding PPP BeiDou especially have been delayed due to other more urgent tasks. I am not sure when I will be able to work more on BeiDou in the close future. I think the earliest I might be able to investigate this more thorough is in August honestly.

Thanks,
I do not know why ppp using BDS in GROOPS performs worse than GPS and even fixes ambiguity incorrectly, since BDS  seems able to reach a relatively good results of float solution, thus I wonder what exactly the problem of this issue? (e.g. solar pressure model, stochastic mode or any other case) Could you give me some clue about that so that we may offer some help in finishing this part in GROOPS. 
Yours sincerely.
Yongzhao Fan

**Reply by zhedumi** [reply]
Yes that is also something that confuses me and I really wish I had the time to investigate it cause it does look interesting. I am not completely sure what the issue might be.
How I planned to approach this problem would be somewhat like this: first read through the master thesis of my collegue who worked on BeiDou for groops [BeiDou Thesis](https://pure.tugraz.at/ws/portalfiles/portal/46400096/Master_Thesis_Oehlinger_Final.pdf)  models he used are on (p 51) etc. for the processing. Cause in his work it looks to be working fine in my oppinion but have not checked much besides the results.
As on how to approach the issue I would recommend first removing the geostationary/geosynchron satellites (these bring more troubles at the moment than benefits) and select a IGS station within china that observes BeiDou. First I would do a float PPP with BeiDou2 then BeiDou3 and check if there is any big differences followed by a combined BeiDou2/3 solution. While doing that I would be checking the logfiles for the redundancies, weighting factor, redundancies and check if some observation type is the issue.
If nothing strikes the eye here I would to the same just with ambiguity fixing and the fixed solution. If the logfiles still do not indicate any issues in the weighting factors etc. I would repeat the same thing with just two selected observation types. If there is still nothing striking the eye I would look into the individual tracks and check how their MW linearcombinations  for the ambiguity fixing look like. If there is still no seeable error I would start going over the models used etc and check if there needs to be something especially considered for BeiDou or some observation type. At least that stuff above was my plan when I have some more time available to check into BeiDou

Best regard

**Reply by FFFFFG-IGG** [reply]
> Yes that is also something that confuses me and I really wish I had the time to investigate it cause it does look interesting. I am not completely sure what the issue might be. How I planned to approach this problem would be somewhat like this: first read through the master thesis of my collegue who worked on BeiDou for groops [BeiDou Thesis](https://pure.tugraz.at/ws/portalfiles/portal/46400096/Master_Thesis_Oehlinger_Final.pdf) models he used are on (p 51) etc. for the processing. Cause in his work it looks to be working fine in my oppinion but have not checked much besides the results. As on how to approach the issue I would recommend first removing the geostationary/geosynchron satellites (these bring more troubles at the moment than benefits) and select a IGS station within china that observes BeiDou. First I would do a float PPP with BeiDou2 then BeiDou3 and check if there is any big differences followed by a combined BeiDou2/3 solution. While doing that I would be checking the logfiles for the redundancies, weighting factor, redundancies and check if some observation type is the issue. If nothing strikes the eye here I would to the same just with ambiguity fixing and the fixed solution. If the logfiles still do not indicate any issues in the weighting factors etc. I would repeat the same thing with just two selected observation types. If there is still nothing striking the eye I would look into the individual tracks and check how their MW linearcombinations for the ambiguity fixing look like. If there is still no seeable error I would start going over the models used etc and check if there needs to be something especially considered for BeiDou or some observation type. At least that stuff above was my plan when I have some more time available to check into BeiDou
> 
> Best regard

Thanks! And I would follow your suggestions and do some examination in Beidou
